app.py

```diff
@@ -36,13 +41,16 @@ def __init__(self):
         decode_button.connect("clicked", self.on_decode_clicked)
 
         self.decoded_label = Gtk.Label()
+        self.decoded_label.set_selectable(True)
```

Comment:
This allows us to select the text so we can copy it and paste if we wanted. Most important for the encode label